test/dockertest.sh

```diff
@@ -31,6 +31,9 @@ while [[ $# -gt 0 ]]; do
     --no-agent)
       DD_NO_AGENT_INSTALL="$2"
       ;;
+    --suse-old)
+      DD_OLD_SUSE="$2"
```

Comment:
suse-old or OLD_SUSE?
I would prefer to have `old-suse`, wdyt?

Reply:
good point 👍